sys-apps/etckeeper/files/etckeeper-1.18.7-gentoo.patch

```diff
@@ -0,0 +1,78 @@
+From 026fa7c012a8c44cadfa56f605c0671639a6b04c Mon Sep 17 00:00:00 2001
```

Comment:
This file is identical to `etckeeper-1.18.6-gentoo.patch`

Reply:
Apparently so, and according to repoman too.

I could refactor the patch naming and associated code if you so desire?